fullstop-web/fullstop-whitelisting-rules/src/main/java/org/zalando/stups/fullstop/controller/RuleController.java

```diff
@@ -107,24 +108,14 @@ public RuleEntity updateWhitelisting(@RequestBody RuleDTO ruleDTO,
         return ruleEntity;
     }
 
-    private boolean hasAccessToAccount(final String userId) {
-        final List<Account> teams = teamOperations.getTeamsByUser(userId);
-        final List<String> allowedTeams = ruleControllerProperties.getAllowedTeams();
 
-        for (Account team : teams) {
-            if (team.getOwner() == null) {
-                continue;
-            }
-            if (allowedTeams.contains(team.getOwner())) {
-                return true;
-            }
-        }
-        return false;
-    }
+    private void checkPermission(String userId) throws ForbiddenException {
+        Preconditions.checkNotNull(userId);
```

Comment:
please add a message (second argument) otherwise we will see "NullPointerException: null" in our logs.